app/resources/v1/announcement_resource.rb

```diff
@@ -12,10 +12,10 @@ class Basechurch::V1::AnnouncementResource < JSONAPI::Resource
   def self.apply_filter(records, filter, value, options)
     case filter
     when :latest_for_group
-      records.where('basechurch_announcements.bulletin_id = ?',
+      records.where('announcements.bulletin_id = ?',
                     self.get_latest_bulletin!(value))
     when :defaults_for_bulletin
-      records.where('basechurch_announcements.bulletin_id = ?',
+      records.where('announcements.bulletin_id = ?',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.